### PR TITLE
Save to PVC for modelcar

### PR DESCRIPTION
**Pull request by RHRolun**

### CODE REVIEW

File: 3-prod_datascience/prod_train_save_pipeline.py [hunk 1]
```diff
@@ -107,6 +107,11 @@ def training_pipeline(hyperparameters: dict, model_name: str, version: str):
         field_path='metadata.namespace'
     )
     register_model_task.after(model_validation_task)
+    kubernetes.mount_pvc(
+        register_model_task,
+        pvc_name=model_strorage_pvc,
```
Comment (rcarrata) [inline 3-prod_datascience/prod_train_save_pipeline.py:112]: typo in model_strorage_pvc, will be model_storage_pvc?

File: 3-prod_datascience/prod_train_save_pipeline.py [hunk 2]
```diff
@@ -115,6 +120,7 @@ def training_pipeline(hyperparameters: dict, model_name: str, version: str):
         },
         "model_name": "jukebox",
         "version": "0.0.2",
+        "model_strorage_pvc": "jukebox-model-pvc",
```
Comment (rcarrata) [inline 3-prod_datascience/prod_train_save_pipeline.py:123]: same in here :)

File: 3-prod_datascience/prod_train_save_pipeline.py (hunk 1 above)
Comment (RHRolun) [reply 3-prod_datascience/prod_train_save_pipeline.py]: Thanks! Updated :)

File: 3-prod_datascience/prod_train_save_pipeline.py (hunk 2 above)
Comment (RHRolun) [reply 3-prod_datascience/prod_train_save_pipeline.py]: Update here too

**Pull request merged**